patches/minecraft/net/minecraft/item/Item.java.patch

```diff
@@ -663,6 +663,19 @@
 +    }
 +
 +    /**
++     * Called to get the Mod ID associated with an ItemStack.
++     * Mods that register subitems for other mods can override this.
++     * @param itemStack the ItemStack to check
++     * @return the Mod ID for the ItemStack
++     */
```

Comment:
[`@return`s should document specialcases](http://www.oracle.com/technetwork/java/javase/documentation/index-137868.html#@return). What does returning `null` mean, exactly?